src/components/views/voip/CallView/CallViewHeader.tsx

```diff
@@ -87,15 +87,15 @@ const CallTypeIcon: React.FC<{ type: CallType }> = ({ type }) => {
 };
 
 const CallViewHeader: React.FC<CallViewHeaderProps> = ({
-    type,
+    type = undefined,
```

Comment:
This should be probably reflected in `CallViewHeaderProps`

Reply:
Now that I think about it some more, I don't like using `undefined` here, could we perhaps use an enum?